server/services/netatmo/lib/netatmo.pollRefreshingValues.js

```diff
@@ -0,0 +1,45 @@
+const { STATUS } = require('./utils/netatmo.constants');
+const logger = require('../../../utils/logger');
+
+/**
+ * @description Poll values of Netatmo devices.
+ * @example refreshNetatmoValues();
+ */
+async function refreshNetatmoValues() {
+  logger.debug('Looking for Netatmo devices values...');
+  await this.saveStatus({ statusType: STATUS.GET_DEVICES_VALUES, message: null });
+
+  let devicesNetatmo = [];
+  try {
+    devicesNetatmo = await this.loadDevices();
+  } catch (e) {
+    await this.saveStatus({
+      statusType: STATUS.ERROR.GET_DEVICES_VALUES,
+      message: 'get_devices_value_fail',
+    });
+    logger.error('Unable to load Netatmo devices', e);
+  }
+  devicesNetatmo.map(async (device) => {
```

Comment:
You cannot do async work in a .map (sync code)

You need to use bluebird Promise.map instead. Don't forget to set a concurrency

Reply:
How to evaluate the competition? Is 10 really suitable? I admit I don't really know. In any case I currently have no lag

Reply:
I would recommended maximum of 2 for something background like this.